transport/server.cc

```diff
@@ -371,6 +371,11 @@ cql_server::connection::read_frame() {
                 _version = current_version;
                 throw exceptions::protocol_exception(format("Invalid or unsupported protocol version: {:d}", client_version));
             }
+
+          auto client_state_notification_f = std::apply(notify_client_change<changed_column::protocol_version>{},
+                  std::tuple_cat(make_client_key(_client_state), std::make_tuple(_version)));
+
+          return client_state_notification_f.then_wrapped([this] (future<>) {
```

Comment:
Shouldn't we check the resulting future for exceptions? Otherwise, we will be sweeping it under the carpet

Reply:
Done